### ISSUE TITLE
New Chapter for San Antonio, TX

### ISSUE THREAD

**Issue by geografa** [maptime/maptime-admin#219]
__Summary__
Maptime name: __MaptimeSA__
Contact name: Luke Whyte
Date submitted: 2017-02-25 23:03:53

- [ ] Add GH admins to new team: https://github.com/lukewhyte
- [ ] Add Location (San Antonio, TX) to chapters.json and /chapters page.

__Additional tasks__
- [ ] Add to Slack
- [ ] set _config.yml file

/cc @maptime/admin-onboarders

**Comment by almccon** [comment]
Hi @lukewhyte! 

Looks like there already is a MaptimeSATX, but maybe it never got off the ground?

http://maptime.io/sanantonio/
http://twitter.com/maptimesatx

From my records, looks like @emilyrbowe might have been trying to start that group.

**Comment by lukewhyte** [comment]
Hey Alan!

I'll reach out to Emily Bowe and see if I can get in touch. Will this issue stay open for me to report back to?

Thanks!

**Comment by almccon** [comment]
Yeah, we can keep it open for now. Also, FYI I found the earlier San Antonio issue here: #149

**Comment by lukewhyte** [comment]
Thanks!

**Comment by emilyrbowe** [comment]
Okay, adding to the proper thread now. Any way we can add @lukewhyte to the San Antonio team so he has access to the repos? Thank you!

**Comment by almccon** [comment]
Invite sent to @lukewhyte, let me know if that works!

**Comment by emilyrbowe** [comment]
Thanks, @almccon!

**Comment by lukewhyte** [comment]
I seem to still be incapable of contributing to "projects". Any thoughts as
to why this might be?

**Comment by geobrando** [comment]
Hey @lukewhyte  Is there anything you need or can I close this ticket
